saleor/graphql/order/filters.py

```diff
@@ -27,7 +28,13 @@
 
 def filter_payment_status(qs, _, value):
     if value:
-        qs = qs.filter(payments__is_active=True, payments__charge_status__in=value)
+        if ChargeStatus.FULLY_REFUNDED in value:
```

Comment:
Shouldn't be `if ChargeStatus.FULLY_REFUNDED == value`?

Reply:
like I commented above. value is a list that can look like this for example: `['fully-refunded', 'partially-refunded']`